TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot_components/robot/DriveBase.java

```diff
@@ -108,31 +108,16 @@ public boolean driveMotorsRunning() {
     }
 
     // Calculates powers for mecanum wheel drive
-    public void calculateDrivePowers(double x, double y, double rot) { // rot is rotation
-        calculateDrivePowers(x, y, rot, false);
+    public void calculateDrivePowers(double x, double y, double r) {
```

Comment:
keep new simple function, make a separate one for meta drive

Reply:
done